Dockerfile

```diff
@@ -151,33 +112,25 @@ RUN pip3 install --editable /nextstrain/augur
 # used for the same reasons described above.
 RUN cd /nextstrain/auspice && npm install && npm run build && npm link
 
+RUN apt-get clean
 
 # ———————————————————————————————————————————————————————————————————— #
 
-
 # Now build the final image.
-FROM alpine:3.7
+FROM python:3.7-slim-buster
 
 # Add system runtime deps
-RUN apk add --no-cache \
+RUN apt-get update
+RUN apt-get install -y \
         ca-certificates \
         curl \
-        bash \
-        freetype \
-        gmp \
         gzip \
-        lapack \
-        libpng \
         nodejs \
-        perl \
-        python2 \
-        python3 \
-        suitesparse \
-        ruby \
-        tar \
```

Comment:
I assume you were able to remove Bash, Perl, Ruby, tar, etc because they're already installed?

If they're not already installed, I do think it'd be useful to keep them so that builds can easily incorporate custom Ruby or Perl scripts, for example.

Reply:
bash, perl, and tar were already in the base image, but ruby was not. I've added ruby back to the list of packages installed in the final image.

Reply:
Great!